src/App.jsx

```diff
@@ -1,8 +1,22 @@
+import { Username } from "./components/Username"
+import { useStore } from "./store/useStore"
+import { StartGame } from "./components/StartGame"
+// import { BrowserRouter, Routes} from "react-router-dom";
+// import { routes } from "./routes/routes";
 
 export const App = () => {
+  const { username } = useStore()
+
   return (
-  <div>
-    Labyrinth Project 
-  </div>
-  );
-};
+    <>
+      {/* <BrowserRouter> */}
+      <div>
+        {!username && <Username />}
+        {username && <StartGame />}
+      </div>
+
+      {/* <Routes>{routes}</Routes>
+  </BrowserRouter> */}
+    </>
```

Comment:
Remove unused code and the fragment and just wrap everything in a div for cleaner code